src/Directory.Packages.props

```diff
@@ -6,30 +6,30 @@
   <ItemGroup>
     <PackageVersion Include="AssertNet" Version="2.0.0" />
     <PackageVersion Include="AsyncFixer" Version="1.6.0" />
-    <PackageVersion Include="AwesomeAssertions " Version="8.2.0" />
-    <PackageVersion Include="AwesomeAssertions.Analyzers" Version="8.0.0" />
-    <PackageVersion Include="BenchmarkDotNet" Version="0.15.0" />
+    <PackageVersion Include="AwesomeAssertions " Version="9.0.0" />
+    <PackageVersion Include="AwesomeAssertions.Analyzers" Version="9.0.0" />
+    <PackageVersion Include="BenchmarkDotNet" Version="0.15.2" />
     <PackageVersion Include="coverlet.collector" Version="6.0.4" />
     <PackageVersion Include="DotNet.ReproducibleBuilds" Version="1.2.25" />
-    <PackageVersion Include="DotNetProjectFile.Analyzers" Version="1.6.1" />
+    <PackageVersion Include="DotNetProjectFile.Analyzers" Version="1.7.0" />
     <PackageVersion Include="Microsoft.CodeAnalysis.Analyzers" Version="4.14.0" />
     <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.14.0" />
     <PackageVersion Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="9.0.0" />
     <PackageVersion Include="Microsoft.Sbom.Targets" Version="4.0.3" />
     <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
-    <PackageVersion Include="Parlot" Version="1.3.6" />
-    <PackageVersion Include="ParsecSharp" Version="4.0.0" />
-    <PackageVersion Include="Pidgin" Version="3.4.0" />
+    <PackageVersion Include="Parlot" Version="1.4.0" />
+    <PackageVersion Include="ParsecSharp" Version="4.1.1" />
+    <PackageVersion Include="Pidgin" Version="3.5.0" />
     <PackageVersion Include="PolySharp" Version="1.15.0" />
     <PackageVersion Include="Qowaiv.Analyzers.CSharp" Version="2.0.5" />
-    <PackageVersion Include="SonarAnalyzer.CSharp" Version="10.10.0.116381" />
+    <PackageVersion Include="SonarAnalyzer.CSharp" Version="10.12.0.118525" />
     <PackageVersion Include="StyleCop.Analyzers.Unstable" Version="1.2.0.556" />
-    <PackageVersion Include="System.Collections.Immutable" Version="1.5.0" />
-    <PackageVersion Include="Verify" Version="30.3.1" />
+    <PackageVersion Include="System.Collections.Immutable" Version="9.0.6" />
```

Comment:
I am very much conflicted about this, because this pulls in a newer version of `System.Collections.Immutable` for .netstandard targets than it does for the .NET 5,6,7,8 targets (where the dependency does not exist). I intentionally targeted the lowest version of `System.Collections.Immutable` that is API compatible with the features being used.

I also know this gives some odd issues with the test and benchmark frameworks (see the weird version override hacks in the csprojs that you have removed).

Reply:
`1.1.37` appears to be an even earlier version, so I'm not entirely sure how this is related

Reply:
Well, they want to support VS2015. That is not the version we should stick to necessarily,